### ISSUE TITLE
Possible to separate free and members' drop tables as a setting?

### ISSUE THREAD

**Issue by jazzyhazzy21** [donth77/loot-lookup-plugin#14]
I've noticed there was a separate entry for the Ankou for members' world drops and f2p world drops, but there isn't anything like that for most enemies. Is it possible to differentiate whether an item is a members' drop only for every enemy and simply not display it if so?

**Comment by donth77** [comment]
@jazzyhazzy21 
That happens because the plugin parses the wiki and the Ankou page has sections titled "Members' world drops" and "Free-to-play" worlds drops.

The wiki also indicates members items for F2P monster drops with "(m)". Would you want this to be reflected in the plugin?

**Comment by jazzyhazzy21** [comment]
@donth77 Yeah, that would be great! Simply appending the "(m)" would work fine. Thank you for your help.
